src/translations/screens/FareContract.ts

```diff
@@ -334,6 +337,9 @@ const FareContractTexts = {
     switch (mode) {
       case TransportMode.Bus:
       case TransportMode.Coach:
+        if (subMode === TransportSubmode.NightBus) {
```

Comment:
@Oleninkaa I think we can remove this, right? since we no longer use this `FareContractTexts`.